gradle-jdks/build.gradle

```diff
@@ -68,5 +69,11 @@ tasks.withType(JavaCompile).configureEach {
 
 tasks.test {
     environment("PROJECT_VERSION", project.version)
-    dependsOn ':gradle-jdks-setup:fatJar'
+    systemProperty 'ignoreDeprecations', 'true'
+}
+
+tasks.withType(ProcessResources.class).configureEach {
```

Comment:
`gradle-jdks-setup.jar` needs to be added to the `gradle/` configuration directory when the JDK setup is enabled. It is used by the patched  `./gradlew` [to run the JDK setup](https://github.com/palantir/gradle-jdks/blob/2f1feae30cad4266dad85d29ba22a47ddaa59a29/gradle-jdks-setup/src/main/resources/gradle-jdks-setup.sh#L154)

The jar will be included in the `gradle-jdks` resources, such that when the `gradle-jdks` plugin is applied, when the  task `wrapperPatcherTask` is called, we can download the jar from the resources and copy it in the 'gradle/' directory. (https://github.com/palantir/gradle-jdks/pull/360/files#diff-664db41a37fb84c50fd2cf1579ff8a17040c6becc39198bb87ecdb468675ddf7R73)

Reply:
adding `.named('processResources')` would do it. Or more generally:

```java
project.getTasks()
        .withType(ProcessResources.class)
        .named(project.getExtensions()
                .getByType(SourceSetContainer.class)
                .getByName(SourceSet.MAIN_SOURCE_SET_NAME)
                .getProcessResourcesTaskName());
```